_posts/2022-01-25-Related-Projects.md

```diff
@@ -56,3 +56,16 @@ within a Docker container. It provides a `Dockerfile` and some documentation.
 [jamulus-server-remote](https://github.com/vdellamea/jamulus-server-remote) provides a lightweight
 web front-end for a headless Jamulus server running on Linux. It allows a user to start and stop recordings,
 and to zip them up and download them via a web browser. It is implemented in PHP.
+
+## Server add-on suite
+
+[These components](https://github.com/orgs/jamulussoftware/discussions/3085) extend the features of a Jamulus server.
+
+* [jamulus-lounge](https://github.com/dtinth/jamulus-lounge) adds a web-based interface that lets people listen to a Jamulus server. It connects to a server using a bot named **lobby**.
+* [gojam](https://github.com/dtinth/gojam) is an implementation of a Jamulus client written in Go. This tool is intended for headless streaming use-cases. Used by **jamulus-lounge**.
+* [jamviz](https://github.com/dtinth/jamviz) shows active connections of a Jamulus server that is running **jamulus-lounge**. Used to show activity to a livestreaming audience.
+
+## Koord
+
+Koord pairs Jamulus with real-time video of participants in one application, available for desktop and mobile devices. The [Koord.live website](https://koord.live/) offers private servers and additional playback and recording features.
```

Comment:
Maybe better say something like Koord provides a Jamulus compatible app. Their app feels quite different to the official one. 

Another thing: To provide equal opportunities, https://melomax.live/ should be mentioned too as Jamulus Server Hoster comparable to Koord.